app/javascript/packs/components/Header.jsx

```diff
@@ -20,32 +20,69 @@ const Logo = () => (
 const useStyles = makeStyles((theme) => ({
   root: {
     backgroundColor: theme.palette.background.paper,
+    transform: "none !important",
     [theme.breakpoints.up("sm")]: {
       paddingLeft: "30px",
       paddingRight: "30px",
     },
   },
+  bar: {
+    [theme.breakpoints.down("sm")]: {
+      flexDirection: "column",
+    },
+  },
   logo: {
     [theme.breakpoints.down("sm")]: {
-      maxWidth: "30vw",
+      maxWidth: "60vw",
       "& > svg": {
         maxWidth: "100%",
       },
     },
   },
+  [theme.breakpoints.down("sm")]: {
+    sticky: {
+      "& > div > div": {
```

Comment:
Could something have a `class` set to avoid needing the CSS path?

Reply:
Good call. I wasn't sure how to reference other classes in CSS-in-JS so this was me being lazy. I'll poke at it a bit more - this is very fragile.